static/system.json

```diff
@@ -940,6 +940,130 @@
+            "packs": [
+                "fall-of-plaguestone-bestiary",
+                "malevolence-bestiary",
+                "menace-under-otari-bestiary",
+                "one-shot-bestiary",
+                "shadows-at-sundown-bestiary",
+                "the-enmity-cycle-bestiary",
+                "the-slithering-bestiary",
+                "troubles-in-otari-bestiary",
+                "night-of-the-gray-death-bestiary",
+                "crown-of-the-kobold-king-bestiary"
+            ]
+        },
+        {
+            "name": "Pathfinder Adventure Paths",
+            "sorting": "m",
+            "packs": [
+                "abomination-vaults-bestiary",
+                "age-of-ashes-bestiary",
+                "agents-of-edgewatch-bestiary",
+                "blood-lords-bestiary",
+                "extinction-curse-bestiary",
+                "fists-of-the-ruby-phoenix-bestiary",
+                "gatewalkers-bestiary",
+                "outlaws-of-alkenstar-bestiary",
+                "kingmaker-bestiary",
+                "quest-for-the-frozen-flame-bestiary",
+                "strength-of-thousands-bestiary",
+                "stolen-fate-bestiary"
+            ]
+        },
+        {
+            "name": "Pathfinder Rulebook Bestiaries",
+            "sorting": "m",
+            "packs": [
+                "pathfinder-bestiary",
+                "pathfinder-bestiary-2",
+                "pathfinder-bestiary-3",
+                "pathfinder-dark-archive",
+                "book-of-the-dead-bestiary",
+                "hazards",
+                "lost-omens-impossible-lands-bestiary",
+                "lost-omens-mwangi-expanse-bestiary",
+                "lost-omens-monsters-of-myth-bestiary",
+                "lost-omens-travel-guide-bestiary",
+                "npc-gallery",
+                "vehicles",
+                "blog-bestiary",
+                "april-fools-bestiary"
+            ]
+        },
+        {
+            "name": "Pregenerated Characters",
+            "sorting": "m",
+            "packs": [
+                "iconics",
+                "paizo-pregens"
+            ]
+        },
+        {
+            "name": "Effects",
+            "sorting": "m",
+            "packs": [
+                "bestiary-effects",
+                "conditionitems",
+                "campaign-effects",
+                "equipment-effects",
+                "feat-effects",
+                "other-effects",
+                "spell-effects"
+            ]
+        },
+        {
+            "name": "Character Building",
+            "sorting": "m",
+            "packs": [
+                "actionspf2e",
+                "ancestries",
+                "ancestryfeatures",
+                "backgrounds",
+                "classes",
+                "classfeatures",
+                "familiar-abilities",
+                "feats-srd",
+                "heritages",
+                "spells-srd",
+                "equipment-srd",
```

Comment:
would probably be good to have equipment unfolder'd since it's just as applicable to NPCs

Reply:
Probably, though the UI looks so awkward with some things out of folders.

Reply:
I feel that the awkwardness may be worth stomaching for the sake of preventing GM confusion.